public/assets/dir.css

```diff
@@ -0,0 +1,321 @@
+html, main, body {
+  background-color: #1E1E1E;
+}
+
+.hidden {
+  display: none;
+}
+
+.dir-upload-wrapper {
+  border-radius: 1rem;
+  padding: 1.5rem 5%;
+}
+
+.container {
+  background-color: #fff;
+  border-radius: 1rem;
+  margin: 0 auto;
+  padding-bottom: 2rem;
+}
+
+.box.is-success .box__success {
+  margin-bottom: 2.5rem;
+}
+
+.box__message > p {
```

Comment:
BEM?

Reply:
The `box__message` was there before I started modifying the file, it's not mine, and I wasn't really worried about following their naming convention at this point - this was just a quick fix. :)